### ISSUE TITLE
errors in --gen3d

### ISSUE THREAD

**Issue by cassguff** [openbabel/openbabel#2676]
## Environment Information
Open Babel version: Open Babel 3.1.1
Operating system and version: Windows 11

## Expected Behavior

I'm just trying to convert a SMILES string to an .xyz file using the --gen3d to get an approximate low energy conformation. I expect to get an xyz file with 3d coordinates.

## Actual Behavior

 It generates the xyz file, but I do not get 3D coordinates ( in that the z column is all 0s) showing that no optimization is happening. I get the following output:

==============================
*** Open Babel Error  in OpenBabel::OBBuilder::LoadFragments
  Cannot open ring-fragments-index.txt
==============================
*** Open Babel Error  in OpenBabel::OBForceFieldMMFF94::ParseParamFile
  Cannot open parameter file
==============================
*** Open Babel Error  in OpenBabel::OBForceFieldUFF::ParseParamFile
  Cannot open UFF.prm
==============================
*** Open Babel Error  in OpenBabel::OBForceFieldUFF::SetTypes
  Cannot open UFF.prm
1 molecule converted

## Steps to Reproduce
I am using the command 'obabel -:"CCCC" -O 1.xyz --gen3D -xb -h' in the command line.

I saw a different thread about someone who added BABEL_DATADIR as an environment variable and fixing a similar problem, but that didn't work for me.

**Comment by nbehrnd** [comment]
@cassguff Based on the [documentation](https://open-babel.readthedocs.io/en/latest/FileFormats/XYZ_cartesian_coordinates_format.html) flag `-b` is an option (`-xb`) of reading.  Your input is a SMILES string and eventually you want to write .xyz coordinates only (i.e., the file has no information about bonds).  An command like

```
obabel -:"c1ccncc1" -h --gen3d -O test.xyz
```

is sufficient for (this example) to generate a conformer of pyridine (including protons for pH 7.4) written into file `test.xyz`.

**Comment by ghutchis** [comment]
The bigger problem is that it's reporting it cannot find any of the force field data files.

You didn't mention how you installed it, but that's the key problem. The install can't find any files needed for coordinate generation.

**Comment by nbehrnd** [comment]
Yeah, https://open-babel.readthedocs.io/en/latest/Installation/install.html

**Comment by cassguff** [comment]
I installed it by simply just downloading the latest version of the 64 bit
binary installer.

Thanks for also pointing out that -b is not needed for this command.

**Comment by cassguff** [comment]
Is there a different way I should download and install?

**Comment by cassguff** [comment]
It didn't work to do the 32 bit installer. I can just use Linux or the
online instance. Thanks.
